pkgs/development/libraries/json-glib/default.nix

```diff
@@ -14,11 +14,14 @@
 , gnome3
 }:
 
+let
+  isCross = stdenv.buildPlatform != stdenv.targetPlatform;
```

Comment:
I would make a `withIntrospection ? stdenv.buildPlatform != stdenv.targetPlatform` parameter and use that instead. I think it was down this way elsewhere FWIW, and it's a bit more self-documenting.

Reply:
Fixed. Thanks for the review :)